app/javascript/components/batch_update/components/Selected/SelectedPrices.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import SelectedComponent from './SelectedComponent'
+
+const SelectedPrices = props => {
+  const { minPrice, maxPrice, clearState } = props
+  return (
+    <div>
+      <h2>Price</h2>
+      {minPrice !== null && (
+        <SelectedComponent
+          className="currentMinPrice"
+          stateKey="minPrice"
+          name={'Minimum: ' + minPrice + ' USD'}
```

Comment:
Sorry one more I forgot to add!

Here and L21, let's do USD before the number. It's both more conventional, and easier to read since the numbers will line up.